docs/_sidebar.md

```diff
@@ -5,8 +5,10 @@
     - [dpp.identity](usage/identity.md)
     - [dpp.stateTransition](usage/stateTransition.md)
 - Primitives
+    - [AbstractStateTransition](primitives/AbstractStateTransition.md)
```

Comment:
Should we list AbstractStateTransition here? It’s kinda abstract class. I guess we need to list just STs that inherited it?

Reply:
That's mostly for type explanation in parameters inputs. We refer to AbstractStateTransition. 
I recon we would want a way to be explicit in different part of them. But in this configuration, yeah we kinda need it.